### ISSUE TITLE
Git merge issue in README?

### ISSUE THREAD

**Issue by croaky** [absinthe-graphql/absinthe_plug#78]
```
def deps do
  [
<<<<<<< HEAD
    ...,
=======
    {:absinthe_plug, "~> 1.2.3"},
>>>>>>> master
    {:poison, "~> 1.3.0"}
  ]
end
```

**Comment by petejkim** [comment]
Hi @croaky, this was fixed in c5a6513526e1f48b0e5cddc5ea7757aed6d98348

**Comment by bruce** [comment]
Thanks for catching this and fixing it. (oops!)
